### DISCUSSION TITLE
Show target branch in PR list

### DISCUSSION THREAD

**Discussion by bendilley** [community/community#7824] [Pull Requests]
When viewing a list of pull requests (open or otherwise), I and my team would find it really useful to be able to see at a glance which branch each PR is targeting.

The ability to filter the list by target branch would also be useful.

**Comment by Arusekk** [comment]
This would be a very useful feature, since currently the only kind of relevant option is to assign milestones to every PR, which is quite tedious (it may be possible to automate, but still).

**Comment by bendilley** [comment]
I notice the [pulls API](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests) does return details of the `base` target in the response, _and_ the base branch name is available as a filter, so I suppose I could roll my own 🤔

**Comment by bendilley** [comment]
Just on the filtering point, I have realised that the query bar allows you to specify e.g. `base:main` as a qualifier to filter down the list, but I'd still really like to see the base listed against each result.

**Comment by wesclements** [comment]
Upvoted. We just moved from Bitbucket to GitHub. Would be nice to display the targets the same way Bitbucket does. Currently, we're using base:qa, base:uat, etc., but it's nice to have an overall view of the targets

**Comment by Karmalakas** [comment]
I can't even find the target anywhere after PR is created. Not even in a PR view.. 🙄

**Comment by tg-dpiscaglia** [comment]
I completely agree, we totally need this please

**Comment by gabechiarelli** [comment]
We're also an ex-Bitbucket team, we sorely miss being able to see the target branch for our branching strategy. I can understand it adding clutter for the average repo but having an option would be nice.

**Comment by dx-bogdanov** [comment]
I work in an ex-GitLab team and miss this feature too.
For now, target branch can be seen in Tooltip, if you hover on PR link long enough.
But this takes time and is especially clumsy when I compare what PRs did I commit into what branches.
So for now I just prefix branch into all my PR names.
